test/go-tests/test_utils.go

```diff
@@ -25,6 +29,56 @@ func (sender *APIEventSender) SendEvent(event v2.Event) error {
 	return err
 }
 
+func EnsureProjectExists(projectName, shipyardFilePath string) error {
```

Comment:
I think this function is not doing what it is called.

1. it checks whether the project exists or not
2. if the project exists, the project is deleted
3. it creates a new project

I think it should be handled differently, e.g., via a param:

```golang
func CreateProject(projectName, shipyardFilePath, recreateIfAlreadyExists boolean) error {
...

	if resp.Response().StatusCode != http.StatusNotFound {
                if (recreateIfAlreadyExists) {
		// delete project if it exists
		_, err = ExecuteCommand(fmt.Sprintf("keptn delete project %s", projectName))
		if err != nil {
			return err
		}
                } else {
                    return error.Error("Project already exists")
                }
	}
...
```